php/commands/sidebar.php

```diff
@@ -73,6 +73,204 @@ public function list_( $args, $assoc_args ) {
 		$formatter->display_items( $wp_registered_sidebars );
 	}
 
+	/**
+	 * Reset one or all sidebars.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<sidebar-id>]
+	 * : ID for the corresponding sidebar.
+	 *
+	 * @subcommand reset
```

Comment:
This definition isn't needed if the command name is the same as the method name.